WalletWasabi.Gui/Converters/LurkingWifeModeStringConverter.cs

```diff
@@ -21,7 +21,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 
 				return new string(Enumerable.Repeat('#', len).ToArray());
 			}
-			return value;
+			return value.ToString();
```

Comment:
What's this?

Reply:
This is because the `LurkingWifeModeStringConverter` needs to return a string, otherwise it only works when the property is string. In this case the `TotalAmount` property is decimal can be bound perfectly but it stops working if the LWMSC is used.

Reply:
**Update**: If I remove this `.ToString()` call then I can change the `TotalAmount` property from `decimal` to `string`. I would like to do that because otherwise we will end having all properties as string.

Reply:
Sounds good.